docs/README.md

```diff
@@ -0,0 +1,47 @@
+# docker-swift
+
+![swift](https://raw.githubusercontent.com/hamin/EventSource.Swift/master/swift-logo.png)
+
+[![Build Status](https://api.shippable.com/projects/58663bf6f6c7101000c4038f/badge?branchName=master)](https://app.shippable.com/projects/58663bf6f6c7101000c4038f/builds/latest)
+
+### An Ubuntu 16.04 Docker image for [Swift](https://swift.org).
+
+#### You can find the Docker Hub repo here: [https://hub.docker.com/r/swiftdocker/swift/](https://hub.docker.com/r/swiftdocker/swift/)
+
+
+### Docker Instructions
+
+##### Pull the Docker Image From Docker Hub:
+
+```bash
+docker pull swiftdocker/swift
+```
+
+##### Create a Container from the Image and Attach It:
+
+```bash
+docker run --privileged -i -t --name swiftfun swiftdocker/swift:latest /bin/bash
```

Comment:
Any particular reason you're including `--privileged` here?  That's essentially giving the container full unfettered access to your host, so definitely not recommended. :sweat_smile: